controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/CommitRequestHandler.java

```diff
@@ -310,7 +310,7 @@ public CommitRequestHandler(final StreamMetadataStore streamMetadataStore,
             // Use fixed scaling policy for these segments as they are created, merged into and sealed and are not
             // supposed to auto scale.
             return streamMetadataTasks.notifyNewSegment(scope, stream, segment, ScalingPolicy.fixed(1), delegationToken, 
-                    context.getRequestId());
+                    context.getRequestId(), 0);
```

Comment:
What segments are these? I suspect these are the internal controller segments. If so, then let's make a GitHub issue to have the Controller set proper value(s) here and reference that issue with a TODO (here and anywhere else that we might need).